backend/src/app/domain/accounts/services.py

```diff
@@ -85,7 +85,7 @@ async def _check_permissions(
         self, db_obj: m.User | None, password: str, hashed_password: str
     ) -> None:
         if hashed_password is None:
-            msg = "User not found or password invalid"
+            msg = "Token not found or already used"
```

Comment:
Might as well update or get rid of https://github.com/MTES-MCT/ecobalyse/blob/ff54d50654168255a07af4c36873ee40197cb494/src/Views/Page.elm#L512-L522